packages/core/src/index.ts

```diff
@@ -805,7 +788,7 @@ class Hopr extends EventEmitter {
   ): Promise<{
     channelId: Hash
   }> {
-    const ethereum = this.paymentChannels
+    const ethereum = this.chain
```

Comment:
Why the simple renaming?

Reply:
I can describe it a bit further, but the name `paymentChannels` is a bit misleading and somewhat outdated. It’s more like our `onchain` adapter, so that’s why I named it like that. Please feel free to suggest a different name.